hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/ddl/TestSpark3DDL.scala

```diff
@@ -706,6 +709,8 @@ class TestSpark3DDL extends HoodieSparkSqlTestBase {
   }
 
   test("Test schema auto evolution") {
+    //This test will be flakey for mor until [HUDI-6798] is landed and we can set the merge mode
```

Comment:
How does the test pass for MOR before?

Reply:
Looks like this test was fixed when the default payload was changed. Now it should work fine with the fg reader